README.md

```diff
@@ -88,20 +91,27 @@ helm -n sparrow upgrade -i sparrow oci://ghcr.io/caas-team/charts/sparrow --vers
 ```
 
 The default settings are fine for a local running configuration. With the default Helm values, the sparrow loader uses a
-runtime configuration that is provided in a ConfigMap. The ConfigMap can be set by defining the `runtimeConfig` section.
+checks configuration that is provided in a ConfigMap (the `file` loader is used). The ConfigMap can be set by defining the `checksConfig` section.
 
-To be able to load the configuration during the runtime dynamically, the sparrow loader needs to be set to type `http`.
+To be able to load the checks configuration during the runtime dynamically, the sparrow loader needs to be set to type `http`.
 
 Use the following configuration values to use a runtime configuration by the `http` loader:
 
 ```yaml
 startupConfig:
-  loaderType: http
-  loaderHttpUrl: https://url-to-runtime-config.de/api/config%2Eyaml
+  ...
+  loader:
+    type: http
+    http:
+      url: https://url-to-checks-config.de/api/config%2Eyaml
 
-runtimeConfig: { }
+checksConfig: {}
 ```
 
+Sensitive data like the token used by the `http` loader (`loader.http.token`) should not be created in the values section. A secret needs to be created manually with a defined environment variable `SPARROW_LOADER_HTTP_TOKEN`. Use the `envFromSecrets` in the `values.yaml` to make the secret accessible by the sparrow container.
```

Comment:
To provide the sparrow container with the token (use active and not passive wording, it's always simpler 😉)